### DISCUSSION TITLE
Better order for user perf stat card

### DISCUSSION THREAD

**Discussion by ZTL-UwU** [lichess-org/mobile#162] [Ideas]
![IMG_20230514_152817.jpg](https://github.com/lichess-org/mobile/assets/49156174/5ad96d83-ee6d-4e5a-a885-5ff1863ed2d9)

The ultra bullet card is not the most important. Yet it is in the first place, blcocking the rapid card.

Maybe we should put the most played variant in front or just put bullet, blitz and rapid in front by default?

**Comment by LukaszSztukiewicz** [comment]
Good point!
